Add new chart for Scalar Admin for Kubernetes (#246)
scalar-labs/helm-charts: docs/how-to-deploy-scalar-admin-k8s.md

# How to deploy Scalar Admin k8s

> This document explains how to deploy Scalar Admin k8s by using Scalar Helm Charts. For details on the custom values file for Scalar Admin k8s, see [Configure a custom values file for Scalar Admin k8s](./configure-custom-values-scalar-admin-k8s.md).

Author: I updated the file path in the documents in 4c52c707aa514ed5ecef0edb5c21c61da3589417 .

## Deploy Scalar Admin k8s

To deploy Scalar Admin k8s, run the following command, replacing the contents in the angle brackets as described:

```console

> helm install <RELEASE_NAME> scalar-labs/scalar-admin-k8s -n <NAMESPACE> -f /<PATH_TO_YOUR_CUSTOM_VALUES_FILE_FOR_SCALAR_ADMIN_K8S> --version <CHART_VERSION>

Reviewer: ```suggestion
helm install <RELEASE_NAME> scalar-labs/scalar-admin-for-kubernetes -n <NAMESPACE> -f /<PATH_TO_YOUR_CUSTOM_VALUES_FILE_FOR_SCALAR_ADMIN_FOR_KUBERNETES> --version <CHART_VERSION>
```

```

> ## Upgrade a Scalar Admin k8s job

Reviewer: ```suggestion
## Upgrade a Scalar Admin for Kubernetes job
```

> To upgrade a Scalar Admin k8s job, run the following command, replacing the contents in the angle brackets as described:

Reviewer: ```suggestion
To upgrade a Scalar Admin for Kubernetes job, run the following command, replacing the contents in the angle brackets as described:
```

```console

> helm upgrade <RELEASE_NAME> scalar-labs/scalar-admin-k8s -n <NAMESPACE> -f /<PATH_TO_YOUR_CUSTOM_VALUES_FILE_FOR_SCALAR_ADMIN_K8S> --version <CHART_VERSION>

Reviewer: ```suggestion
helm upgrade <RELEASE_NAME> scalar-labs/scalar-admin-for-kubernetes -n <NAMESPACE> -f /<PATH_TO_YOUR_CUSTOM_VALUES_FILE_FOR_SCALAR_ADMIN_FOR_KUBERNETES> --version <CHART_VERSION>
```

```

> ## Delete a Scalar Admin k8s job

Reviewer: ```suggestion
## Delete a Scalar Admin for Kubernetes job
```

> To delete a Scalar Admin k8s job, run the following command, replacing the contents in the angle brackets as described:  

Reviewer: ```suggestion
To delete a Scalar Admin for Kubernetes job, run the following command, replacing the contents in the angle brackets as described:  
```
